src/utils/hendelseFilteringUtils.tsx

```diff
@@ -63,3 +64,37 @@ export const filterEventsOnVeileder = (personregister: PersonregisterState, veil
     }, {} as PersonregisterState);
     return final;
 };
+
+export type SortingType = 'NAME_ASC' | 'NAME_DESC' | 'NONE';
+
+export const getSortedEventsFromSortingType = (personregister: PersonregisterState, type: SortingType) => {
+    if (type === 'NAME_DESC') {
+        return sortEventsOnName(personregister, 'NAME_DESC');
+    } else if (type === 'NAME_ASC') {
+        return sortEventsOnName(personregister, 'NAME_ASC');
+    }
+    return personregister;
+};
+
+export const sortEventsOnName = (personregister: PersonregisterState, order: SortingType): PersonregisterState => {
+    const sortingResultAsArray = Object.keys(personregister).reduce((currentState, fnr) => {
```

Comment:
Har splittet denne opp for å indikere hva hver enkelt funksjon gjør/returnerer da jeg følte det gav litt bedre forståelse av de forskjellige stegene som inngår i sorteringen.